src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,27 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
```

Comment:
user can be undefined. do not render this block in this case
```suggestion
      {post.user && (
      <p>
        {' Posted by  '}

        <UserInfo user={post.user} />
      </p>)
    }
```